docs/reference/commandline/dockerd.md

```diff
@@ -780,6 +785,113 @@ conditions the user can pass any size less then the backing fs size.
 $ sudo dockerd -s overlay2 --storage-opt overlay2.size=1G
 ```
 
+
+#### Windowsfilter options
+
+##### `size`
+
+Specifies the size to use when creating the sandbox which is used for containers.
+Defaults to 20G.
+
+###### Example
+
+```PowerShell
+C:\> dockerd --storage-opt size=40G
+```
+
+#### LCOW (Linux Containers on Windows) options
+
+##### `lcow.globalmode`
```

Comment:
Are the `lcow` options merged with the container's `--storage-opt` now (sorry if a silly question, wasn't sure if `runtime.GOOS == "windows"` is / is not true for LCOW)